src/OrchardCore/OrchardCore.Users.Core/Services/ConfigureSecurityStampOptions.cs

```diff
@@ -0,0 +1,35 @@
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Options;
+
+namespace OrchardCore.Users.Services;
+
+public class ConfigureSecurityStampOptions : IPostConfigureOptions<SecurityStampValidatorOptions>
+{
+    public void PostConfigure(string name, SecurityStampValidatorOptions options)
```

Comment:
You should add a comment explaining the reason for this handler.

Reply:
Good idea. I'll add comments in the next day or so.